backend/app/app/templates/profile.html

```diff
@@ -188,15 +314,80 @@ <h5 class="modal-title" id="hours-modal-{{ sem.semester_id }}-title">
 
                 <div class="container portrait">
                     <div class="col">
-                        <div class="row justify-content-center">
+                      <div class="row justify-content-center">
+                          <div class="col-md-6 col-lg-6 col-xl-6 col-left">
+
                             {% if student.is_ill %}
-                                <button type="button" class="btn btn-dark" onclick="toggle_ill(this)">Recovered &#x1F601
-                                </button>
-                            {% else %}
-                                <button type="button" class="btn btn-dark" onclick="toggle_ill(this)">Got sick &#x1F912
-                                </button>
-                            {% endif %}
+                          <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+                           Recovered  &#x1F601
+                          </button>
```

Comment:
Think of making this button `btn-dark` instead of `btn-primary` in order to make it distinguishable from group choice button